### PR TITLE
[Docs] Additional guides/troubleshoot installation links in quickstart

### LINKED ISSUE #5598: [Docs] Quick Start: Need link to Installation Troubleshooting
#### Current State
The bottom of the Quick Start (https://docs.meshery.io/installation/quick-start) page ends without links to additional Guides.

#### Desired State
The bottom of the Quick Start (https://docs.meshery.io/installation/quick-start) page could use a Resources section with link to https://docs.meshery.io/guides/troubleshooting/installation and https://docs.meshery.io/reference/error-codes

and could use a description of or pointer to `mesheryctl system check`

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by abhiyant-10**
**Description**

This PR fixes #5598

**Notes for Reviewers**
Added additional guides in docs/pages/installation/quick-start.md
Fixed error codes link  that is not working on https://docs.meshery.io/guides/troubleshooting 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
Signed by : Abhiyant Gwalani <abhigwalani@gmail.com>
 - [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 96e52148066c6e0a1db7e3812ba7f46fc47e3cdc at: https://62aba371d5721137a1da786a--meshery-docs-preview.netlify.app

File: docs/pages/installation/quick-start.md [hunk 1]
```diff
@@ -83,3 +83,10 @@ You may now proceed to install and work with any [service mesh]({{ site.baseurl
  Your browser does not support the video tag
 </video>
 
+
+## Additional Guides
```
Comment (leecalcote) [inline docs/pages/installation/quick-start.md:87]: It would be good to pre-pend the site base URL variable before each of these hyperlinks

**Comment by leecalcote** [comment]
@abhiyant-10 will you rename this PR to be descriptive of the issue?

File: docs/pages/installation/quick-start.md (hunk 1 above)
Comment (abhiyant-10) [reply docs/pages/installation/quick-start.md:87]: Sure ! I did in both the ways . Sorry , I was not able to push the new changes I will create a new PR.

**Pull request closed without merging**